code/src/main/java/com/onlinecontacttracing/servlets/delete-data/PositiveUserPlacesDeleter.java

```diff
@@ -0,0 +1,60 @@
+package com.onlinecontacttracing.servlets;
+
+import static com.googlecode.objectify.ObjectifyService.ofy;
+import com.onlinecontacttracing.storage.PositiveUserPlaces;
+import com.onlinecontacttracing.storage.Constants;
+import com.google.common.collect.Iterables;
+import com.googlecode.objectify.Key;
+import java.util.logging.Logger;
+import java.time.Instant;
+
+/**
+* This class deletes old notification batches
+*/
+class  PositiveUserPlacesDeleter extends OldDataDeleter {
+
+  public Class getType() {
+    return PositiveUserPlaces.class;
+  }
+
+  public String getQueryString() {
+    return "timeCreatedSeconds <";
+  }
+
+  /**
+  * To calculate the time for querying old data, this method
+  * subtracts maximum time a class is allowed to persist in memory
+  * from the current time by the 
+  */
+  public long getMaxAge() {
```

Comment:
If you modify this method to be "getRetentionDuration", you could just return the constant value, without doing Instant.now() here. You could do it in the abstract class.

Reply:
done